### DISCUSSION TITLE
Have the possibility to have a FLAC library and transcode all for simple playing with slow network

### DISCUSSION THREAD

**Discussion by yodatak** [navidrome/navidrome#3009] [Ideas]
Hello thanks for navidrome, I love the transcoding but I would love to have the possibility to transcode my library in FLAC in différents bitrate to permit simple streaming with low network and don't need to live transcode

**Comment by deluan** [comment]
Navidrome does not pre-transcode your files, but transcoded files are cached, so they should only be transcoded once. You can increase the size of the cache by setting the [TranscodingCacheSize](https://www.navidrome.org/docs/usage/configuration-options/#:~:text=(disable%20TLS)-,TranscodingCacheSize,-ND_TRANSCODINGCACHESIZE) option.
